server/views/layout/patterns.njk

```diff
@@ -14,16 +14,7 @@
-</svg>
+{% include "views/partials/svg-sprite.njk" %}
```

Comment:
Why are we keeping this file out of the build?
It would mean that we have a hard dependency on the client compile step to run the server?